packages/manager/modules/adp/src/deploy/review.html

```diff
@@ -0,0 +1,138 @@
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="admin_mail" data-ng-bind="$ctrl.adp.admin_mail"></span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- nodes config -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_nodes_title"></h6>
+        <div class="border-top">
+            <form>
+                <div class="form-group row" data-ng-repeat="(nodeType, node) in $ctrl.nodesConfig track by nodeType">
+                    <label for="{{ ::nodeType }}_nodes"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_nodes_{{ ::nodeType }}">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="{{ ::nodeType }}_nodes">
+                            {{ ::node.count }} x {{ node.selectedFlavor.displayName }}
+                        </span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- storage config -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_storage_title"></h6>
+        <div class="border-top">
+            <form>
+                <div class="form-group row">
+                    <label for="raw_storage"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_storage_raw">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="raw_storage" data-ng-bind="$ctrl.adp.hdfs_effective_storage"></span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="usable_storage"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_storage_usable">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="usable_storage" data-ng-bind="$ctrl.adp.edge_node_storage"></span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- price details -->
+    <div class="mb-4">
+        <div class="row border rounded border-secondary bg-light">
+            <div class="col-xs-12 col-sm-5 m-2">
+                <section>
+                    <h6 data-translate="adp_deploy_review_hourly_price"></h6>
+                    <p data-translate="adp_common_review_hourly_price_format"
+                        data-translate-values="{price: $ctrl.price.hourlyPrice, currency: '€'}"></p>
```

Comment:
Currency couldn't be retrieve from the API?

Sounds strange to hardcoded this.

Reply:
This is in our to-do list. We will do it as a part of another PR